client/header/activity-panel/panels/inbox/cards.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import { applyFilters } from '@wordpress/hooks';
+import { Icon, box, comment, page } from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Bell } from './icons/bell';
+
+const EXTENDED_TASK_LIST_ID = 'extended_task_list';
+const ORDER_PANEL_ID = 'orders-panel';
+const REVIEWS_PANEL_ID = 'reviews-panel';
+const STOCK_PANEL_ID = 'stock-panel';
+
+export const cards = applyFilters( 'woocommerce_admin_abbreviated_card_list', [
```

Comment:
See comment below about filters vs. SlotFill.

Reply:
I removed this filter in the commit 9c729f4